service/src/main/kotlin/fi/espoo/evaka/decision/DecisionDraftService.kt

```diff
@@ -28,6 +28,7 @@ class DecisionDraftService {
             PlacementType.DAYCARE, PlacementType.DAYCARE_PART_TIME -> planDaycareDecisionDrafts(placementPlan)
             PlacementType.PRESCHOOL, PlacementType.PRESCHOOL_DAYCARE, PlacementType.PREPARATORY, PlacementType.PREPARATORY_DAYCARE -> planPreschoolDecisionDrafts(placementPlan, application)
             PlacementType.TEMPORARY_DAYCARE, PlacementType.TEMPORARY_DAYCARE_PART_DAY,
+            PlacementType.SCHOOL_SHIFT_CARE -> listOf()
```

Comment:
This probably doesn't matter, but doesn't this change the semantics of temporary daycares?

Previously `TEMPORARY_DAYCARE`, `TEMPORARY_DAYCARE_PART_DAY`, `DAYCARE_FIVE_YEAR_OLDS`, `DAYCARE_PART_TIME_FIVE_YEAR_OLDS` all ended up throwing errors.

Now `TEMPORARY_DAYCARE`, `TEMPORARY_DAYCARE_PART_DAY` return an empty list

Reply:
Fixed